scss/forms/_star-rating.scss

```
@@ -3,31 +3,30 @@
 //
 
 .star-rating {
-  &.disabled,
-  &:disabled {
-    pointer-events: none;
-  }
 
-  > label {
+  > label,
+  span {
     display: block;
     float: left;
     width: 1em;
     height: 1em;
     font-size: $form-star-size;
     cursor: pointer;
-    @include form-star-rating($accessible-orange);
+    @include form-star-rating($accessible-orange, $accessible-orange);
 
     &:not(:first-of-type){
       margin-left: $form-star-margin-between;
     }
   }
 
-  > input:checked ~ label {
-    @include form-star-rating($gray-500);
+  > input:checked ~ label,
+  span {
+    @include form-star-rating($gray-700, $white);
   }
 
-  > input:checked + label {
-    @include form-star-rating($accessible-orange);
+  > input:checked + label,
+  .checked {
```

```suggestion
  span.checked {
```
if the above one is commited

---

Not sure about this one, it will just cause a 'selector-no-qualifying-type' error 🤔

---

Ah you're right the rule above had more effect than expected. Actually wondering if this line is still needed

---

Yes, we don't need it any more.